cluster-autoscaler/cloudprovider/oci/README.md

```diff
@@ -166,6 +166,13 @@ use-instance-principals = true
 
 n/a
 
+### Node Group Auto Discovery
+`--node-group-auto-discovery` could be given in below pattern. It would discover the nodepools under given compartment by matching the nodepool tags (either they are Freeform or Defined tags)
+```
+clusterId:<clusterId>,compartmentId:<compartmentId>,nodepoolTags:<tagKey1>=<tagValue1>&<tagKey2>=<tagValue2>,min:<min>,max:<max>
```

Comment:
Are all of the fields in this required? Or are any optional? Can we specify in the comment string above?

Reply:
all of the fields are mandatory. I've added a statement to make it clear in readme.